Bottom: 206bdcf50a445429ccd0d18f7c81a195ff63f2b5
Top:    cbebfa2b63cb7d5c56a0a3c79e43af01e9711edd
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-20 10:41:17 -0500

Add script to provision staging server


---
diff --git a/provision.sh b/provision.sh
new file mode 100755
index 00000000..3762e89d
--- /dev/null
+++ b/provision.sh
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+#PYTHONUNBUFFERED=1 \
+     
+ANSIBLE_ENABLE_TASK_DEBUGGER=True \
+ANSIBLE_FORCE_COLOR=true \
+ANSIBLE_HOST_KEY_CHECKING=false \
+ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ControlMaster=auto -o ControlPersist=60s' \
+exec ansible-playbook \
+        --connection=ssh \
+        --timeout=30 \
+        --inventory-file=ansible-playbook/staging \
+        -vv \
+        ansible-playbook/site.yml "$@"
